### PR TITLE
Add technical properties section to aid in migration.

### PR DESCRIPTION
**Pull request by markpbaggett**
What Does this Do?
==================

Adds a technical properties section to aid in migration.

How Should This Be Tested?
==========================

Do tests pass? Does rationale make sense? Are sections complete enough?

Additional Notes
================

This closely relates to [the Digital Collections MAP](https://docs.google.com/spreadsheets/d/1_0QVbQU_wj3ITUih5dGPGkWHN0QyhGO9hKSf6rXwKPc/edit#gid=0).

The frame height / width on the sample don't exist yet, but will after approval.

### CODE REVIEW

File: docs/source/contents/5_technical_metadata_properties.rst [hunk 1]
```diff
@@ -0,0 +1,245 @@
+####################
+Technical Properties
+####################
+
+*****
+About
+*****
+
+While our MODS to RDF mapping aims to address how descriptive properties should be mapped in our next system, this
+section aims to explain how to map existing technical metadata properties our metadata properties that we need going
```
Comment (mlhale7) [inline docs/source/contents/5_technical_metadata_properties.rst]: I think you're missing a "to"? Should it say "map existing technical metadata properties to our metadata properties"?

File: docs/source/contents/5_technical_metadata_properties.rst [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+forward.  When a technical metadata property may exist, its current mapping is described.
```
Comment (mlhale7) [inline docs/source/contents/5_technical_metadata_properties.rst]: I'm not sure what you mean by "When a technical metadata property may exist, its current mapping is described." The "may exist" portion is confusing to me. If there's a current mapping and it exists, why say "may"?

File: docs/source/contents/5_technical_metadata_properties.rst [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+Mapping
+*******
+
+Duration
+========
+
+Use Case
+--------
+
+For audio / visual files, it is essential that we are able to measure the length of a the file that is associated with a
```
Comment (mlhale7) [inline docs/source/contents/5_technical_metadata_properties.rst]: Extra word on this line - get rid of "a".

File: docs/source/contents/5_technical_metadata_properties.rst [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+
+Frame Height
+============
+
+Use Case
+--------
+
+Video files do not have a standard height across the repository, but height is important for display in certain viewers.
+
+Height is unecessary for images because that should be supplied by the image server.
```
Comment (mlhale7) [inline docs/source/contents/5_technical_metadata_properties.rst]: Missing "n" - unnecessary

File: docs/source/contents/5_technical_metadata_properties.rst [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+Frame Width
+===========
+
+Use Case
+--------
+
+Video files do not have a standard width across the repository, but width is important for display in certain viewers.
+
+Width is unecessary for images because that should be supplied by the image server.
```
Comment (mlhale7) [inline docs/source/contents/5_technical_metadata_properties.rst]: Missing "n" - unnecessary

**Review by mlhale7 — CHANGES_REQUESTED** [review]

File: docs/source/contents/5_technical_metadata_properties.rst (hunk 2 above)
Comment (markpbaggett) [reply docs/source/contents/5_technical_metadata_properties.rst]: 👍🏻

File: docs/source/contents/5_technical_metadata_properties.rst (hunk 3 above)
Comment (markpbaggett) [reply docs/source/contents/5_technical_metadata_properties.rst]: 👍🏻

**Review by mlhale7 — APPROVED** [review]

**Pull request merged**